packages/common/src/lib/common/tools.js

```diff
@@ -475,7 +475,7 @@ async function createUuid(objects) {
     const promiseCheckPassword = new Promise(resolve =>
         objects.getObject('system.user.admin', (err, obj) => {
             if (err || !obj) {
-                password = password || require('./password');
+                password = password || require('./password').password;
```

Comment:
Any reason we can't statically import `./password` at the top?

Reply:
bf style